jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/RadioTests.java

```diff
@@ -0,0 +1,54 @@
+package io.github.epam.material.tests.inputs;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.Test;
+
+import static com.epam.jdi.tools.LinqUtils.safeException;
+import static io.github.com.StaticSite.*;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.testng.Assert.fail;
+
+public class RadioTests extends TestsInit {
+
+    @Test
+    public void defaultRadioTest() {
+        radioDefaultPage.open();
+
+        radioFrame.firstRadio.is().enabled();
+        radioFrame.secondRadio.is().enabled();
+        radioFrame.thirdRadio.is().enabled();
+        radioFrame.disabledRadio.is().disabled();
+
+        radioFrame.firstRadio.is().unchecked();
+        radioFrame.secondRadio.is().unchecked();
+        radioFrame.thirdRadio.is().unchecked();
+        radioFrame.disabledRadio.is().unchecked();
+
+        radioFrame.firstRadio.toggle();
+        radioFrame.firstRadio.is().checked();
+        radioFrame.secondRadio.is().unchecked();
+        radioFrame.thirdRadio.is().unchecked();
+        radioFrame.disabledRadio.is().unchecked();
+        radioFrame.textField.is().text(containsString("First"));
+
+        radioFrame.secondRadio.toggle();
+        radioFrame.firstRadio.is().unchecked();
+        radioFrame.secondRadio.is().checked();
+        radioFrame.thirdRadio.is().unchecked();
+        radioFrame.disabledRadio.is().unchecked();
+        radioFrame.textField.is().text(containsString("Second"));
+
+        try {
```

Comment:
Please create new test for try/catch block. As example you could check io.github.epam.material.tests.inputs.FabTests#disabledFabTest